libphobos/m4/druntime/libraries.m4

```diff
@@ -57,10 +57,91 @@ AC_DEFUN([DRUNTIME_LIBRARIES_ZLIB],
 
 # DRUNTIME_LIBRARIES_ATOMIC
 # -------------------------
-# Allow specifying whether to use libatomic for
-# 128 bit atomic support.
 AC_DEFUN([DRUNTIME_LIBRARIES_ATOMIC],
 [
+  # Check for atomic builtins up to 64 bit.
```

Comment:
`druntime/cpu.m4` or `druntime/builtins.m4` is probably a better place for this, but either way is fine by me.